packages/shared/src/components/banners/common.ts

```diff
@@ -1,6 +1,9 @@
 import classed from '../../lib/classed';
 
 export const bottomBannerBaseClasses =
-  'fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center';
+  'fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center border-none';
```

Comment:
This will affect all banner cases?
Maybe it breaks something with announcement banner?

Reply:
Seems to only be referenced in the two banners altered in this commit, so I think we should be good

![Screenshot 2024-11-21 at 10 46 23](https://github.com/user-attachments/assets/c1cd2cfc-8cf3-47ff-9e5e-1a7203b48ef7)

Reply:
Cool cool